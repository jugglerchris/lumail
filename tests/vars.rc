local testvar = require('testutil').testvar

testvar(history_file, 'foo')
testvar(index_limit, 'foo2')
--md_prefix = maildir_prefix()
testvar(maildir_prefix, "/tmp/")