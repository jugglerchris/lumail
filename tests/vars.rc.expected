
foo
all
foo2
nil
/tmp/
Exit: 0
